keps/1224-lending-limit/README.md

```diff
@@ -0,0 +1,120 @@
+# KEP-1224: Lending limit
+
+- [Summary](#summary)
+- [Motivation](#motivation)
+  - [Goals](#goals)
+  - [Non-Goals](#non-goals)
+- [Proposal](#proposal)
+  - [User Stories (Optional)](#user-stories-optional)
+    - [Story 1](#story-1)
+  - [Risks and Mitigations](#risks-and-mitigations)
+- [Design Details](#design-details)
+  - [Kueue LendingLimit API](#Kueue LendingLimit API)
+  - [Test Plan](#test-plan)
+    - [Prerequisite testing updates](#prerequisite-testing-updates)
+    - [Unit Tests](#unit-tests)
+    - [Integration tests](#integration-tests)
+  - [Graduation Criteria](#graduation-criteria)
+- [Implementation History](#implementation-history)
+- [Drawbacks](#drawbacks)
+- [Alternatives](#alternatives)
+
+## Summary
+
+This proposal provides a guaranteed resource quota for user. By setting `lending limit`, users can have a reservation of resource quota(`nominalQuota` - `lendingLimit`) that will never be borrowed by other clusterqueues in the same cohort.
```

Comment:
Let's try to not mention as much technical details as possible, just clarify what we hope to do here, like:
```
In today's Kueue's system, one clusterQueue's resources could be borrowed completely by neighbors inside the same cohort, 
this improves the resource utilization to some extend, but sometimes users want to reserve some resources only for private usage. 
This proposal introduces a new field helps to solve this problem by reserving the demand resources.
```